stable/grafana/values.yaml

```diff
@@ -17,6 +17,12 @@ downloadDashboardsImage:
   tag: latest
   pullPolicy: IfNotPresent
 
+## Pod Annotations
+# podAnnotations: {}
+
+## Deployment annotations
+# annotations: {}
```

Comment:
Can you also add these into the README.md?

Reply:
Added, thanks.